ui/src/domain/index.ts

```diff
@@ -11,160 +11,8 @@ export interface Env {
   };
```

Comment:
Since we're now following a split domain pattern, I'm tempted to finish the job and create the modules `env.ts` and `json.ts`

Reply:
The splitting of the domain comes with the drawback of increasing the number of import lines on other modules.

Would you consider maintaining `src/domain/index.ts` and re-export all the inner domains from there?

Like this, we'd keep things organized in different modules and still, we could import every domain entity from `src/domain`.

Reply:
Sure, LGTM.

Reply:
Cool. Merged.